packages/@react-spectrum/test-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-spectrum/test-utils",
-  "version": "3.0.0-alpha.1",
+  "version": "1.0.0-alpha.1",
```

Comment:
won't this be *more* confusing? with all the @react-spectrum/@react-aria scopes being 3.*?

Reply:
yeah, a bit torn on this (and there wasn't a earlier conversation, just something I noticed when auditing haha). I kinda wanted to reset and follow the 1.* version pattern since the 3.* didn't really make sense since it implies that there was a 1.*/2.*